Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java

```diff
@@ -28,39 +31,39 @@
 /**
  * NoteActivity implements activity to show and edit note in a full window view.
  */
-public class NoteActivity extends Activity implements ColorDialogFragment.ColorChangedListener{
+public class NoteActivity extends Activity implements ColorDialogFragment.ColorChangedListener {
     public final static String TAG = "NoteActivity";
 
-    private Note        mNote;
-    private Note.Type   mNoteType;
-    private Intent      mShareIntent;
-    private boolean     mSaveButtonClicked = false; /* To avoid action on reclicking  */
-    private boolean     mAutoSaveEnabled;
-    private boolean     mLoadingNote = true;    /* Is note loaded from database? */
+    private Note mNote;
+    private Note.Type mNoteType;
+    private Intent mShareIntent;
+    private boolean mSaveButtonClicked = false; /* To avoid action on reclicking  */
+    private boolean mAutoSaveEnabled;
+    private boolean mLoadingNote = true;    /* Is note loaded from database? */
```

Comment:
[CR0] Our style guide likes horizontal alignment at class level at-least. Regardless, style changes to unrelated code should be different commit for cleaner diffs.